azuredeploy.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
There are 2 way to deploy Image:
- create web app on Linux and set docker container on azure
- deploy image by [ARM](https://deploy.azure.com) where **azuredeploy.json** is used

We also specify default docker image in it